manifests/params.pp

```diff
@@ -27,6 +28,10 @@
         $netgroup_default   = ['nisplus']
       }
 
+      if $facts[operatingsystemmajrelease] == '8' {
+        $file_path = '/etc/authselect/nsswitch.conf'
```

Comment:
I don't think that this is correct.

If you're using Puppet, you want to authoritatively manage the file and ignore whatever `authselect` madness happens on the system.